schema/spacewalk/upgrade/susemanager-schema-4.4.6-to-susemanager-schema-4.4.7/002-store-vulnerable-packages.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE IF NOT EXISTS suseOVALVulnerablePackage
```

Comment:
this table looks to be duplicated from the file 001-oval-add-tables.sql

Reply:
Nice catch, this should be fixed now.